packages/trpc-playground/src/get-default-input.ts

```diff
@@ -25,32 +25,33 @@ export const getProcedureSchemas = (procedures: Procedures) => {
   const procedureTypes: ProcedureTypes = { queries: {}, mutations: {} }
 
   Object.entries(procedures)
-    .filter(([, { _def }]) => _def.query || _def.mutation)
+    .filter(([, { _def }]) => _def.type === 'query' || _def.type === 'mutation')
     .forEach(([procedureName, procedure]) => {
-      const inputParser = getInputFromInputParsers(procedure._def.inputs)
-      if (typeof inputParser === 'function') {
-        return z.any()
-      }
-
-      const defaultInputValue = inputParser ? getDefaultForDef(inputParser._def) : ''
-
-      let procedureType = ''
-      let docsType = ''
-
-      if (inputParser) {
-        const { node } = zodToTs(inputParser)
-        procedureType = `input: ${printNode(node)}`
-
-        docsType = printNode(createTypeAlias(node, 'input', inputParser.description))
-      }
+      // const inputParser = getInputFromInputParsers(procedure._def.inputs)
+      // if (typeof inputParser === 'function') {
+      //   return z.any()
+      // }
+      //
+      // const defaultInputValue = inputParser ? getDefaultForDef(inputParser._def) : ''
+      const defaultInputValue = ''
+
+      const procedureType = ''
+      const docsType = ''
+
+      // if (inputParser) {
+      //   const { node } = zodToTs(inputParser)
+      //   procedureType = `input: ${printNode(node)}`
+      //
+      //   docsType = printNode(createTypeAlias(node, 'input', inputParser.description))
+      // }
```

Comment:
Can you look into this please?

https://gist.github.com/andrewgeorgemitchell/33f3a97ce3d4d8312b71834a940091d8#file-trpc-playground-fix-ts-L139

Found from comment:
https://github.com/sachinraja/trpc-playground/issues/55#issuecomment-1965824213

Reply:
It seems like all of the changes in that gist should be in this PR. Thoughts?